app/views/home/_bookkeeper_office_manager.html.erb

```diff
@@ -0,0 +1,66 @@
+	</p>
+	<h5>About this Role</h5>
+	<p>
+		Girl Develop It is seeking a highly organized, experienced full-time Bookkeeper/Office Manager to provide administrative support to the organization’s central operations and to support the organization’s financial functions including budget development, income tax preparation, and charitable and business registration filings and renewals. The ideal candidate will have advanced experience in financial management for a nonprofit with rapid growth and multiple revenue streams, demonstrated professionalism in communications, and strong organizational and time management skill with acute attention to detail. We envision an enthusiastic professional who exemplifies our values of stewardship, integrity, community, and wants to part of a growing organization — with understanding that it’s necessary to be flexible to thrive in a fast paced environment. Experience working remotely or on a distributed team is highly desired.
+	</p>
+	<p>
+		This position will report directly to our Operations Director, and work very closely with other members of senior management and chapter leadership teams, providing crucial support to our 53 chapters across the country. Position can be performed remotely upon satisfactory completion of in-person orientation period.
+	</p>
+	<h5>Duties &amp; Responsibilities</h5>
+	<p>
+		Bookkeeping
+	<ul>
+		<li>Running AP/AR, invoicing, general ledger, account and bank reconciliation</li>
+		<li>Processing, as well as tracking all event payments, sponsorships, and scholarships</li>
+		<li>Preparing and distributing monthly financial statements</li>
+		<li>Administering payroll and benefits</li>
+		<li>Managing contracts and vendor relations</li>
+		<li>Developing organization and project budgets in coordination with senior management and leadership teams</li>
+		<li>Supporting annual audit, tax preparation, and annual filings for charitable registrations</li>
+		<li>Maintaining bookkeeping best practices and efficient, transparent systems for collecting, managing, and distributing funds throughout the organization</li>
+	</ul>
+	</p>
+	<p>
+		Administration
+	<ul>
+		<li>Providing day-to-day office administration, handling a wide range of administrative and executive support-related tasks</li>
+		<li>Providing operational support to Chapter Leaders</li>
+		<li>Coordinating meetings and logistics for annual events: galas and leadership summit</li>
+	</ul>
+	</p>
+
+	<h5>Requirements</h5>
+	<p>
+		As a prerequisite, the successful candidate must believe in Girl Develop It's core values and be driven by the mission. Additionally, the following experiences and skills are required:
```

Comment:
The apostrophe in "Girl Develop It's" is the right one if you're looking for a visual example :)